sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClient.java

```diff
@@ -823,44 +750,79 @@ public String getEntityPath() {
     }
 
     /**
-     * Asynchronously renews the lock on the specified message. The lock will be renewed based on the setting specified
+     * Asynchronously renews the lock on the message. The lock will be renewed based on the setting specified
      * on the entity. When a message is received in {@link ReceiveMode#PEEK_LOCK} mode, the message is locked on the
-     * server for this receiver instance for a duration as specified during the Queue creation (LockDuration). If
+     * server for this receiver instance for a duration as specified during the entity creation (LockDuration). If
      * processing of the message requires longer than this duration, the lock needs to be renewed. For each renewal, the
      * lock is reset to the entity's LockDuration value.
      *
-     * @param lockToken Lock token of the message to renew.
+     * @param message The {@link ServiceBusReceivedMessage} to perform auto-lock renewal.
      *
      * @return The new expiration time for the message.
-     * @throws NullPointerException if {@code lockToken} is null.
+     * @throws NullPointerException if {@code message} is null.
      * @throws UnsupportedOperationException if the receiver was opened in {@link ReceiveMode#RECEIVE_AND_DELETE}
      *     mode.
      * @throws IllegalStateException if the receiver is a session receiver.
-     * @throws IllegalArgumentException if {@code lockToken} is an empty value.
+     * @throws IllegalArgumentException if {@code message.getLockToken()} is an empty value.
      */
-    public Mono<OffsetDateTime> renewMessageLock(String lockToken) {
+    public Mono<OffsetDateTime> renewMessageLock(ServiceBusReceivedMessage message) {
```

Comment:
Why are we using `ServiceBusReceivedMessage` for renewing and other settlement methods? Should we use the locktoken instead as per this [API view](https://apiview.dev/Assemblies/Review/6af850ed7eb04843a6897afbe13cdd28#com.azure.messaging.servicebus.ServiceBusReceiverAsyncClient)?

Reply:
@srnagar  I should update this APIview with this change.  There was a review with Clemens (Messaging Architect ) and Service team on this . The requirement of  using lockToken was dropped. Here is discussion thread https://github.com/Azure/azure-sdk-for-net/issues/10852#issuecomment-679859788   Here is an issue for this https://app.zenhub.com/workspaces/azure-sdk-team-5bdca72c4b5806bc2bf0aab2/issues/azure/azure-sdk-for-java/14664

Reply:
The API view has been updated already: https://apiview.dev/Assemblies/Review/64f40be9988242c9a8facc8a20c5e9e7